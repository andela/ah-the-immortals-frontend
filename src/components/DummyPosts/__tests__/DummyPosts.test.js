import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Post from '../DummyPosts';

describe('Tests dummy post component', () => {
  const wrapper = shallow(
    <Post />
  );
  it('tests the homepage component', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
