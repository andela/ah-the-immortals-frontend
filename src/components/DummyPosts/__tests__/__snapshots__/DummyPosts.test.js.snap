// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests dummy post component tests the homepage component 1`] = `
<div>
  <DammyNav />
  <br />
  <br />
  <div
    className="container"
  >
    <br />
    <div
      className="row"
    >
      <div
        className="col"
      >
        <div
          className="list-group"
        >
          <Link
            className="list-group-item list-group-item-action flex-column align-items-start active"
            to="#null"
          >
            <div
              className="d-flex w-100 justify-content-between"
            >
              <h5
                className="mb-1"
              >
                Es6 and beyond
              </h5>
              <small>
                3 days ago
              </small>
            </div>
            <p
              className="mb-1"
            >
              Donec id elit non mi porta gravida at eget metus. Maecenas sed diam eget risus varius blandit.
            </p>
            <small>
              Donec id elit non mi porta.
            </small>
          </Link>
          <Link
            className="list-group-item list-group-item-action flex-column align-items-start"
            to="/"
          >
            <div
              className="d-flex w-100 justify-content-between"
            >
              <h5
                className="mb-1"
              >
                Async await
              </h5>
              <small
                className="text-muted"
              >
                1 day ago
              </small>
            </div>
            <p
              className="mb-1"
            >
              Donec id elit non mi porta gravida at eget metus. Maecenas sed diam eget risus varius blandit.
            </p>
            <small
              className="text-muted"
            >
              Donec id elit non mi porta.
            </small>
          </Link>
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="list-group"
        >
          <Link
            className="list-group-item list-group-item-action flex-column align-items-start active"
            to="/"
          >
            <div
              className="d-flex w-100 justify-content-between"
            >
              <h5
                className="mb-1"
              >
                Baba yaga
              </h5>
              <small>
                3 days ago
              </small>
            </div>
            <p
              className="mb-1"
            >
              Lorem Ipsum is simply deen the industrys standard dummy text ever since the 1500s, when an unk diam eget risus varius blandit.
            </p>
            <small>
              Donec id elit non mi porta.
            </small>
          </Link>
          <Link
            className="list-group-item list-group-item-action flex-column align-items-start"
            to="/"
          >
            <div
              className="d-flex w-100 justify-content-between"
            >
              <h5
                className="mb-1"
              >
                List group item heading
              </h5>
              <small
                className="text-muted"
              >
                2 days ago
              </small>
            </div>
            <p
              className="mb-1"
            >
              Donec id elit non miLorem ince the 1500s, when an unkndiam eget risus varius blandit.
            </p>
            <small
              className="text-muted"
            >
              Donec id elit non mi porta.
            </small>
          </Link>
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="list-group"
        >
          <Link
            className="list-group-item list-group-item-action flex-column align-items-start active"
            to="/"
          >
            <div
              className="d-flex w-100 justify-content-between"
            >
              <h5
                className="mb-1"
              >
                It is okay to cry
              </h5>
              <small>
                3 days ago
              </small>
            </div>
            <p
              className="mb-1"
            >
              Donec id elit non mi porta gravida at eget metus. Maecenas sed diam eget risus varius blandit.
            </p>
            <small>
              Donec id elit non mi porta.
            </small>
          </Link>
          <Link
            className="list-group-item list-group-item-action flex-column align-items-start"
            to="/"
          >
            <div
              className="d-flex w-100 justify-content-between"
            >
              <h5
                className="mb-1"
              >
                List group item heading
              </h5>
              <small
                className="text-muted"
              >
                3 days ago
              </small>
            </div>
            <p
              className="mb-1"
            >
              It has survived not only five centuries, but also the leap into electroMaecenas sed diam eget risus varius blandit.
            </p>
            <small
              className="text-muted"
            >
              Donec id elit non mi porta.
            </small>
          </Link>
        </div>
      </div>
    </div>
  </div>
</div>
`;
